docs/docs/cleaning.md

```diff
@@ -0,0 +1,203 @@
+# Cleaning (Experimental)
+
+While using `nbautoexport`, you may sometimes end up with leftover files you no longer want. Some ways this can happen are if you rename notebooks, or if you change your export configuration. The `nbautoexport` CLI has an experimental `clean` command to delete extraneous files.
+
+**WARNING: the `clean` command can delete files irreversibly. Please use with care.**
+
+## Basic usage
+
+Let's say you have a project and keep your notebooks in a `notebooks/` subdirectory. You can clean it with:
+
+```bash
+nbautoexport clean notebooks/
+```
+
+Note that in order to use `clean`, this directory _must_ be configured with `nbautoexport`, i.e., you have a `notebooks/.nbautoexport` configuration file.
+
+If you have some files that would be cleaned, you would see something that looks like this:
+
+```text
+Identified following files to clean up:
+  notebooks/html/0.1-ejm-data-exploration.html
+  notebooks/html/0.2-ejm-feature-creation.html
+  notebooks/script/Untitled.py
+Are you sure you want to delete these files? [y/N]: █
+```
+
+At this point, you can enter `y` to continue with the deletion or `N` to cancel the deletion. There is also a `--dry-run` flag to automatically exit at this point without prompting to continue with deletion.
+
+## How it works
+
+Let's say you have the following files:
+
+```text
+notebooks/
+├── .ipynb_checkpoints
```

Comment:
I'd consider removing the .ipynb_checkpoints in all these examples for readability/clarity since they're not critical in understand the `clean` functionality. Thoughts?